app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -318,19 +318,24 @@ private void setAvatar(String url,
 
     }
 
-    protected void setCreatedAt(Date createdAt, StatusDisplayOptions statusDisplayOptions) {
+    protected void setCreatedAt(Date createdAt, Date editedAt, StatusDisplayOptions statusDisplayOptions) {
+        String timestampText;
         if (statusDisplayOptions.useAbsoluteTime()) {
-            timestampInfo.setText(absoluteTimeFormatter.format(createdAt, true));
+            timestampText = absoluteTimeFormatter.format(createdAt, true);
         } else {
             if (createdAt == null) {
-                timestampInfo.setText("?m");
+                timestampText = "?m";
             } else {
                 long then = createdAt.getTime();
                 long now = System.currentTimeMillis();
                 String readout = TimestampUtils.getRelativeTimeSpanString(timestampInfo.getContext(), then, now);
-                timestampInfo.setText(readout);
+                timestampText = readout;
             }
         }
+
+        if (editedAt != null)
```

Comment:
I'd say if you don't mind use the brackets {} as the project uses them everywhere, even for one line if statements (at least that has been my limited experience). 

There's more instances of this, so I will not spam you with comments, but I'm sure the project maintainer will care for the consistency.